src/tests/App.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import App from "../components/App";
+
+describe("App", () => {
+  test("renders App component correctly", () => {
+    const { asFragment } = render(<App />);
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
Its a convention to include a line of white space before the expect assertions just to separate the file up a bit. Not critical, and standards may change across companies but either way its good to be consistent across all of your tests